web/gds-admin-ui/src/components/TopbarSearch.js

```diff
@@ -1,67 +1,69 @@
-                            />
-                            <div className="w-100">
-                                <h5 className="m-0 font-14">
-                                    {option.userDetails.firstname} {option.userDetails.lastname}
-                                </h5>
-                                <span className="font-12 mb-0">{option.userDetails.position}</span>
-                            </div>
-                        </div>
-                    </Link>
-                </>
-            );
-
-        default:
-            return;
-    }
-};
+// const optionGetter = (option) => {
+//     switch (option.type) {
+//         case 'report':
+//             return (
+//                 <Link to="/" className={classNames('dropdown-item', 'notify-item', 'p-0')}>
+//                     <i className={classNames(option.icon, 'font-16', 'me-1')}></i>
+//                     <span>{option.label}</span>
+//                 </Link>
+//             );
+//         case 'help':
+//             return (
+//                 <Link to="/" className={classNames('dropdown-item', 'notify-item', 'p-0')}>
+//                     <i className={classNames(option.icon, 'font-16', 'me-1')}></i>
+//                     <span>{option.label}</span>
+//                 </Link>
+//             );
+//         case 'settings':
+//             return (
+//                 <Link to="/" className={classNames('dropdown-item', 'notify-item', 'p-0')}>
+//                     <i className={classNames(option.icon, 'font-16', 'me-1')}></i>
+//                     <span>{option.label}</span>
+//                 </Link>
+//             );
+//         case 'title':
+//             return (
+//                 <div className="noti-title">
+//                     <h6 className="text-overflow mb-2 text-uppercase">Users</h6>
+//                 </div>
+//             );
+//         case 'users':
+//             return (
+//                 <>
+//                     <Link to="/" className="dropdown-item notify-item p-0">
+//                         <div className="d-flex">
+//                             <img
+//                                 src={option.userDetails.avatar}
+//                                 alt=""
+//                                 className="d-flex me-2 rounded-circle"
+//                                 height="32"
+//                             />
+//                             <div className="w-100">
+//                                 <h5 className="m-0 font-14">
+//                                     {option.userDetails.firstname} {option.userDetails.lastname}
+//                                 </h5>
+//                                 <span className="font-12 mb-0">{option.userDetails.position}</span>
+//                             </div>
+//                         </div>
+//                     </Link>
+//                 </>
+//             );
+
+//         default:
+//             return;
+//     }
+// };
```

Comment:
I think we can probably get rid of this once we're ready to move forward - it's unlikely that we'll have multiple types of options in the near future, maybe just some icons and we can reference the original code right? Let me know if you're uncomfortable removing this code for some other reason.

Reply:
I just forgot to remove it